instagram-app/src/components/SearchBar/SearchBar.js

```diff
@@ -1,12 +1,27 @@
 import React from "react";
+import "./SearchBar.scss";
 
-const SearchBar = props => {
-  // console.log(props.);
-  return (
-    <div className="search-bar">
-      <input type="text" className="input" placeholder="Search..." />
-    </div>
-  );
-};
-
+class SearchBar extends React.Component {
+  render() {
+    return (
+      <nav className="Nav">
+        <div className="Nav-menus">
+          <div className="Nav-brand">
+            <a className="Nav-brand-logo" href="/">
+              Instagram
+            </a>
+          </div>
+          <input
+            name="search"
+            type="text"
+            onChange={this.props.searchFilter}
+            className="input"
+            // value={this.props.newSearch}
```

Comment:
```suggestion
            value={this.props.newSearch}
```
You're right, this works commented out, but leave it in anyway. Read this on why https://reactjs.org/docs/forms.html#controlled-components